docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -60,13 +57,13 @@ $ newgrp libvirtd
 [[kvm-driver-fedora]]
 === On Fedora
 
-.  Install `libvirt` and `qemu-kvm` on your system:
+.  Install *libvirt* and *qemu-kvm* on your system:
```

Comment:
hmm, not sure whether one should consider libvirt and qemu-kvm as actual binaries.